lab_6-7/lab6-7-index-opt-3.md

```diff
@@ -591,19 +631,25 @@ GROUP BY Name, Surname
 | --------- | ----------------------- |
 | 1         | ![](img/ex4/query5.png) |
 | 2         | ![](img/ex4/query6.png) |
-| 3         | ![](img/ex4/query7.png) |
 
 **Plany wykonania**
 
 | Zapytanie | Plan wykonania              |
 | --------- | --------------------------- |
 | 1         | ![](img/ex4/query5plan.png) |
 | 2         | ![](img/ex4/query6plan.png) |
-| 3         | ![](img/ex4/query7plan.png) |
+
+**Rekomendacja Databse Engine Tuning Advisor**
+
+![](img/ex4/recommend3.png)
+
+> Narzędzie sugeruje utworzeniu indeksu, który został przez nas już zdefiniowany, czyli indeks wykorzystujący kilka atrybutów.
+
+> Wniosek: Indeksy wykorzystujące kilka atrybutów, wraz z include są przydatne, gdy zapytania obejmują wiele kolumn w warunkach wyszukiwania zapytania.
```

Comment:
powtórzenie - "zapytania [...] w warunkach zapytania"

Reply:
Refactored this text